benchmarks/serialization_libraries/compare.sh

```diff
@@ -0,0 +1,222 @@
+#   },                                                              },
+#   "id": "5",                                                      "id": "5",
+#   "relationships": {                                              "relationships": {
+#     "comments": {                                                   "comments": {
+#       "data": [                                                       "data": [
+#         {                                                               {
+#           "id": "6",                                                      "id": "6",
+#           "type": "comments"                                              "type": "comments"
+#         },                                                              },
+#         {                                                               {
+#           "id": "7",                                                      "id": "7",
+#           "type": "comments"                                              "type": "comments"
+#         }                                                               }
+#       ]                                                               ]
+#     },                                                              },
+#     "user": {                                                       "user": {
+#       "meta": {                                               |       "data": {
+#         "included": false                                     |         "id": "1",
+#                                                               >         "type": "users"
+#       }                                                               }
+#     }                                                               }
+#   },                                                              },
+#   "type": "posts"                                                 "type": "posts"
+# }                                                               }
+# -------------------------------------included comments
+# {                                                               {
+#   "attributes": {                                                 "attributes": {
+#     "author": "me",                                                 "author": "me",
+#     "comment": "nice blog"                                          "comment": "nice blog"
+#   },                                                              },
+#   "id": "3",                                                      "id": "3",
+#   "relationships": {                                              "relationships": {
+#     "post": {                                                       "post": {
+#       "meta": {                                               |       "data": {
+#         "included": false                                     |         "id": "2",
+#                                                               >         "type": "posts"
```

Comment:
jsonapi-rb excludes the comments.relationships.post.data since comments are included via post.comments